### PR TITLE
[stable/Jenkins] Update plugins version

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Update plugins to avoid deprecated errors and update docs:

jenkins-plugin-error

### CODE REVIEW

**Comment by BrunoQuaresma** [comment]
CLA did!

File: stable/jenkins/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.16.16
+version: 0.16.17
```
Comment (davidkarlsen) [inline stable/jenkins/Chart.yaml]: Bump this one more step so we can merge https://github.com/helm/charts/pull/6987 then this one

**Comment by BrunoQuaresma** [comment]
@davidkarlsen I did it!

**Pull request merged**